explore-analyze/visualize/charts/tables.md

```diff
@@ -0,0 +1,264 @@
+:   Define the formatting and behavior of each metric column, including:
+    
+    - **Name**: The column header label. By default, the chart uses the function or formula name. It's a best practice to customize this with a meaningful title.
+    
+    :::{include} ../../_snippets/lens-tables-column-appearance-settings.md
+    :::
+    
+    - **Summary row**: Add a row at the bottom of the table showing an aggregate value for this column. You can choose the aggregation function (`Sum`, `Average`, `Min`, `Max`, `Count`) and customize the **Summary label**.
+
+### Rows settings [rows-options]
+
+**Data**
+:   Define which fields create the rows of your table. Drag a field to the **Rows** dimension, and {{kib}} suggests an appropriate function based on the field type. 
+
+    - **Functions**:
+      - **Top values**: Show the most common values of a categorical field. Configure the number of values to display, ranking criteria, and sort direction.
+        - **Number of values**: How many top values to display
+        - **Rank by**: Which metric to use for ranking
+        - **Rank direction**: Ascending or descending order
+      - **Date histogram**: Group data by time intervals. Configure the time interval and how to handle date formatting.
+      - **Intervals**: Create numeric ranges for continuous data. Useful for grouping numeric fields into buckets.
+      - **Filters**: Define custom KQL filters to create specific row groups. Each filter creates one row in the table.
+
+    - **Collapse by**: Aggregate rows that share the same value for this field into a single row, combining their metrics (for example, sum or average for each group). This is useful when you want to display a consolidated result for grouped values instead of individual rows.
+
+    :::{include} ../../_snippets/lens-breakdown-advanced-settings.md
+    :::
+
+**Appearance**
+:   - **Name**: Customize the column header label for the row dimension.
+    :::{include} ../../_snippets/lens-tables-column-appearance-settings.md
+    :::
+    - **Directly filter on click**: Make the values in this column clickable, so clicking a value adds a filter to your visualization or dashboard for that value. This interactivity is helpful for quickly drilling down into data.
+
+### Split metrics by settings [columns-options]
+
+**Data**
+:   Optionally split your metrics into separate columns based on a categorical field. This creates a pivot table view where each unique value of the split field becomes its own column. This is useful for comparing the same metric across different categories side by side.
+
+    You can configure:
+    - **Number of values**: How many columns to create
+    - **Rank by**: Which metric determines the top values
+    - **Rank direction**: Sort order for selecting top values
+
+    :::{include} ../../_snippets/lens-breakdown-advanced-settings.md
+    :::
+
+**Appearance**
+:   - **Name**: Customize the split dimension. This name is not used on the table.
```

Comment:
If not used, is it worth mentioning it?

Reply:
I would argue why having this setting at all :D